components/cam/src/control/cam_comp.F90

```diff
@@ -166,8 +164,7 @@ subroutine cam_init( cam_out, cam_in, mpicom_atm, &
 
    else
 
-      call cam_read_restart ( cam_in, cam_out, dyn_in, dyn_out, pbuf2d, &
```

Comment:
Good call on removing the state and tend. I forgot to remove these when I took out the 2-way tendency mapping for the physgrid. Shouldn't matter for the move to E3SM though.